### PR TITLE
Formal fixes

### PR DESCRIPTION
**Pull request by ligi**
Currently working on a CAIP checker that can check headers for correctness - this removes some inconsistencies

### CODE REVIEW

File: CAIPs/caip-25.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ status: Draft
 type: Standard
 created: 2020-10-14
 updated: 2022-10-26
-requires: ["2", "10"]
+requires: 2, 10
```
Comment (bumblefudge) [inline CAIPs/caip-25.md:10]: It appears the `"`s were unnecessary, but the `[]`s *were* necessary.  Not sure if this is all Yaml, or the specific version of Yaml Jekyll uses, or just Jekyll, but:

https://talk.jekyllrb.com/t/solved-front-matter-array-with-multiple-values/4426

File: CAIPs/caip-3.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Blockchain Reference for the EIP155 Namespace
 author: Simon Warta (@webmaster128), ligi <ligi@ligi.de>, Pedro Gomes (@pedrouid)
 discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/3, https://github.com/ChainAgnostic/CAIPs/pull/1
 status: Draft
```
Comment (bumblefudge) [inline CAIPs/caip-3.md:6]:
```suggestion
status: Superseded
```

Comment (bumblefudge) [reply CAIPs/caip-3.md:6]: (Embarrassing! I'll cop to it, it was me that changed the wrong property to Superseded haha)

File: CAIPs/caip-25.md (hunk 1 above)
Comment (ligi) [reply CAIPs/caip-25.md:10]: OK then the other way around - but for now we had both styles which was not good:

CAIPs/CAIPs/caip-25.md:requires: ["2", "10"]
CAIPs/CAIPs/caip-76.md:requires: 10, 75

File: CAIPs/caip-3.md (hunk 2 above)
Comment (ligi) [reply CAIPs/caip-3.md:6]: if this is a real state we need to add it to CAIP-1

**Review by bumblefudge — APPROVED** [review]

File: CAIPs/caip-25.md (hunk 1 above)
Comment (bumblefudge) [reply CAIPs/caip-25.md]: Fixed!

Comment (bumblefudge) [reply CAIPs/caip-25.md]: And a comment added to CAIP-1 and to the template.md

**Pull request merged**